modules/eco-poison-pill-operator-installation-web-console.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * /machine_management/eco-poison-pill-operator.adoc
+
+[id="installing-poison-pill-operator-using-web-console_{context}"]
+= Installing the Poison Pill Operator using the web console
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware using installer-provisioned infrastructure (IPI).
+* Install the OpenShift CLI (`oc`).
```

Comment:
Would the user need access to the CLI if they are using the Web Console? Or would they just need access to the Web Console?

Reply:
Corrected - they would need the web console